docs/src/SUMMARY.md

```diff
@@ -1,25 +1,34 @@
 # The Kani Rust Verifier
 
-- [Getting started with Kani](./getting-started.md)
+# Getting started
+
+  - [Introduction](./introduction.md)
   - [Installation](./install-guide.md)
-  - [Comparison with other tools](./tool-comparison.md)
-  - [Kani on a single file](./kani-single-file.md)
-  - [Kani on a package](./cargo-kani.md)
-  - [Kani result types](./kani-result-types.md)
-  - [Debugging failures]()
-  - [Debugging non-termination]()
-  - [Debugging coverage]()
+  - [Usage](./usage.md)
```

Comment:
I don't see the need for this page, or putting the below in a sublist.

Reply:
I think splitting into subsections here (and in the next case) provides a much cleaner and organized look to our documentation. In my opinion, the previous version was a little "too flat", so adding subsections where we can may help with readability and navigation.